package Statement;

public class elseif {
    public static void main(String[] args) {
        int age = 20;
        if (age >= 18) {
            System.out.println("Adult");
        } else if (age >= 13) {
            System.out.println("teenager");
        } else {
            System.out.println("child");
        }
    }
}
